src/_tests/integration/integration-utils.js

```diff
@@ -27,12 +27,23 @@ const waitForNoSpinners = page => {
   return page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
 }
 
+const delay = ms => {
+  return new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
TOL: There's an implementation of this in `utils.js`, but I think making a copy here is a good choice for now, since we're still figuring out all the ways this will get packaged up and operationalized. So this keeps things a bit more self-contained.